contracts/linear/src/validator_pool.rs

```diff
@@ -743,7 +743,11 @@ impl LiquidStakingContract {
             .unwrap_or_else(|| panic!("{}: {}", ERR_VALIDATOR_NOT_EXIST, &validator_id));
 
         if is_promise_success() {
-            validator.on_unstake_success(&mut self.validator_pool, amount);
+            validator.on_unstake_success(
```

Comment:
Should we also sync balance for `drain_unstake()`?

Reply:
Sure. I mean. Why not?